### PR TITLE
`make update-common` to pick up recent changes

### PR DESCRIPTION
**Pull request by ericvn**
**Please provide a description of this PR:**

Fix the lint error

Fixes: https://github.com/istio/istio/issues/43000

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Comment by ericvn** [comment]
Looks like it needs changes from https://github.com/istio/istio/pull/42760 as well.

File: pilot/pkg/model/context.go [hunk 1]
```diff
@@ -472,17 +471,13 @@ func (s *StringBool) UnmarshalJSON(data []byte) error {
...
-		return nil, err
-	}
-	return buf.Bytes(), nil
+	return protomarshal.Marshal(pc)
 }
 
 func (s *NodeMetaProxyConfig) UnmarshalJSON(data []byte) error {
 	pc := (*meshconfig.ProxyConfig)(s)
-	return jsonpb.Unmarshal(bytes.NewReader(data), pc)
+	return protomarshal.UnmarshalAllowUnknown(data, pc)
```
Comment (ericvn) [inline pilot/pkg/model/context.go:480]: I notice this has AllowUnknown. Not sure if that's an issue or not.

**Comment by ericvn** [comment]
Making this DNM since I added more changes to get this common-files update to merge and I don't want @howardjohn approval to necessarily let them merge.

Still chasing a few small items.

**Comment by ericvn** [comment]
Removing DNM since I'm not going to try and cherry-pick another PR. I'm just disabling the dependency guard in gloang-ci in this branch.

**Pull request merged**